test/virtualization.test.ts

```diff
@@ -227,23 +213,14 @@ test('zero columns', () => {
 test('zero rows', () => {
   setupGrid(true, 20, 0);
 
-  assertHeightFill(1080 - rowHeight + 35);
   expect(getHeaderCells()).toHaveLength(18);
   expect(queryCells()).toHaveLength(0);
   expect(queryRows()).toHaveLength(0);
 });
 
-test('zero rows + 2 summary rows', () => {
-  setupGrid(true, 20, 0, 0, 2);
-
-  assertHeightFill(1080 - rowHeight * 3 + 35 + 70);
```

Comment:
I wonder if it's worth testing the values of `--rdg-summary-row-bottom`/`--rdg-summary-row-top`

Reply:
I will check in the next PR